### ISSUE TITLE
How to save the mesh of the current pose state of shadowHands

### ISSUE THREAD

**Issue by Running-Chen** [JenniferBuehler/graspit-pkgs#65]
Hi, thanks for your excellent work. I wonder if it is possible to save the mesh of the current pose state of shadow hands?

**Comment by JenniferBuehler** [comment]
No, there is no method to save the mesh. You could record the joint states, which would lead to that pose.
Not sure this helps, or maybe I misunderstood the question ;)
